samples/system-test/v1p1beta1/notifications.test.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {SecurityCenterClient} =
+    require('@google-cloud/security-center').v1p1beta1;
+const {assert} = require('chai');
+const {describe, it, before} = require('mocha');
+const {execSync} = require('child_process');
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+
+const organizationId = process.env['GCLOUD_ORGANIZATION'];
+const orgName = 'organizations/' + organizationId;
+const pubsubTopic = 'projects/project-a-id/topics/notification-sample-topic';
+
+describe('Client with Notifications', async () => {
+  const createConfig = 'notif-config-test-node-create';
+  const deleteConfig = 'notif-config-test-node-delete';
+  const getConfig = 'notif-config-test-node-get';
+  const listConfig = 'notif-config-test-node-list';
+  const updateConfig = 'notif-config-test-node-update';
+
+  before(async () => {
+    const client = new SecurityCenterClient();
+    async function createNotificationConfig(configId) {
+      const [response] = await client.createNotificationConfig({
+        parent: orgName,
+        configId: configId,
+        notificationConfig: {
+          description: 'Sample config for node.js',
+          pubsubTopic: pubsubTopic,
+          eventType: 'FINDING',
+          streamingConfig: {filter: 'state = "ACTIVE"'},
+        },
+      });
+    }
+
+    createNotificationConfig(deleteConfig);
+    createNotificationConfig(getConfig);
+    createNotificationConfig(listConfig);
+    createNotificationConfig(updateConfig);
+  });
+
+  after(async () => {
+    const client = new SecurityCenterClient();
+    async function deleteNotificationConfig(configId) {
+      const name = client.notificationConfigPath(organizationId, configId);
+      await client.deleteNotificationConfig({name: name});
+    }
+
+    deleteNotificationConfig(createConfig);
+    deleteNotificationConfig(getConfig);
+    deleteNotificationConfig(listConfig);
+    deleteNotificationConfig(updateConfig);
+  });
+
+  it('client can create config', () => {
+    const output = exec(`node v1p1beta1/createNotificationConfig.js ${
+        organizationId} ${createConfig} ${pubsubTopic}`)
+    assert.match(output, new RegExp(createConfig));
```

Comment:
This could be `assert.include(output, createConfig)` to keep it a little more simple

Reply:
Done